pom.xml

```diff
@@ -114,6 +114,18 @@
       <version>4.4.6</version>
       <type>jar</type>
     </dependency>
+    <dependency>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>3.1.6</version>
```

Comment:
Why do we use this particular version? Latest version is 4.0.2.

Reply:
Updated